src/cockpit/beiboot.py

```diff
@@ -324,54 +335,69 @@ def do_init(self, message):
 async def run(args) -> None:
     logger.debug("Hi. How are you today?")
 
-    bridge = SshBridge(args)
-    StdioTransport(asyncio.get_running_loop(), bridge)
-
-    try:
-        message = dict(await bridge.ssh_peer.start())
-
-        # See comment in do_init() above: we tell cockpit-ws that we support
-        # this and then handle it ourselves when we get the init message.
-        capabilities = message.setdefault('capabilities', {})
-        if not isinstance(capabilities, dict):
-            bridge.write_control(command='init', problem='protocol-error', message='capabilities must be a dict')
+    with tempfile.TemporaryDirectory() as tmpdir:
+        known_hosts_file = Path(tmpdir) / 'user-known-hosts'
```

Comment:
@allisonkarlitskaya This large indented `with` block is ugly. I know how much you love contexts :grin: I'm also happy to move the temp dir creation into `__init__`, then we can drop that large indentation again.

Reply:
I love contexts but I don't love large blocks of excessive indentation (or noisy patches).  Feel free to hang this on the `SshPeer`, instead...

Reply:
Moved to SshPeer.